lib/ClangImporter/ClangImporter.cpp

```diff
@@ -6197,9 +6199,11 @@ TinyPtrVector<ValueDecl *> ClangRecordMemberLookup::evaluate(
           continue;
 
         // Add Clang members that are imported lazily.
-        auto baseResults = evaluateOrDefault(
-            ctx.evaluator,
-            ClangRecordMemberLookup({cast<NominalTypeDecl>(import), name}), {});
+        auto baseResults =
```

Comment:
So `baseResults` includes all the members from all the bases recursively here?

Reply:
It includes all the results inherited from only `import` here. This is inside of a loop that iterates through all base classes, so `baseResults` will contain the results for each base class (and its ancestors) separately.